cos-fleetshard-operator/src/main/java/org/bf2/cos/fleetshard/operator/operand/OperandResourceWatcher.java

```diff
@@ -49,6 +55,11 @@ public OperandResourceWatcher(
 
     }
 
+    @Override
+    public String name() {
```

Comment:
is this used anywhere ?

Reply:
Yes. The SDK cannot register two EventSources with the same name, and the default implementation returns the class name. Since we register multiple EventSources of this type, we need a different name for each instance.